### PR TITLE
Add ec council

### PR DESCRIPTION
**Pull request by stephengroat**

### CODE REVIEW

**Comment by stephengroat** [comment]
I really want to add CompTIA too, but there website login vs cert login is confusing.

I'll try to find someone who has Security+ to test

**Review by fpigerre — APPROVED** [review]

File: _data/education.yml [hunk 1]
```diff
@@ -51,6 +51,13 @@ websites:
       twitter: duolingo
       tfa: No
 
+    - name: EC-Council
+      url: https://www.eccouncil.org
+      img: eccouncil.png
+      tfa: Yes
+      sms: Yes
+      doc: https://aspen.eccouncil.org/aspen/Help/2FactorAuthentication-User_Manual.pdf
```
Comment (Jawshy) [inline _data/education.yml:60]: Is the documentation available outside of a PDF file? I have been under the assumption that links to PDF files are to be avoided if at all possible.

Comment (stephengroat) [reply _data/education.yml:60]: couldn't find anything else

Comment (Jawshy) [reply _data/education.yml:60]: The documentation also specifies sending the verification code via email. See page 2 and 3. 

> 1. ...
> 2. ...
> 3. Select one of the options presented on the screen. Select My mobile number to receive
>    the verification code on your mobile number, My registered email to receive the ... verification code on your registered email, or No, I want to use a backup code if you have
>    a backup verification code, and click Use this method. 

Could you please add the following? 

``` ruby
      email: Yes
```

**Review by Jawshy — CHANGES_REQUESTED** [review]
`email` TFA.

**Review by Jawshy — APPROVED** [review]

**Pull request merged**